src/server/core/GoogleApiForwarder.ts

```
@@ -1,40 +1,251 @@
-    const generativeModel = genAI.getGenerativeModel({ model: modelId });
+  private maxProxyRetries: number = 1;
+  private requestTimeout: number = 30000; // 30 seconds
+  private rotatingProxyFailureCount: number = 0;
+  private maxRotatingProxyFailures: number = 3;
+  private rotatingProxyDisabledUntil?: number;
+  private healthMonitor?: RotatingProxyHealthMonitor;
+  private healthMonitor?: RotatingProxyHealthMonitor;
+  
+  /**
+   * Set the maximum number of proxy retries
+   */
```

Biome (1.9.4)

[error] Duplicate class member name "healthMonitor"



(lint/suspicious/noDuplicateClassMembers)